package-lock.json

```diff
@@ -618,7 +618,7 @@
     },
     "array-equal": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
```

Comment:
to specjalnie? ;)

Reply:
Kwestia wywolania `npm install sakiewka` - nie wiem czemu tam sie dzieja takie zmiany od czasu do czasu. :(